.github/workflows/go-tests.yml

```diff
@@ -516,3 +516,14 @@ jobs:
             printf "Tests failed or workflow cancelled:\n\n${{ toJSON(needs) }}"
             exit 1
           fi
+      - name: Notify Slack
+        if: ${{ failure() && (github.ref_name == 'main' || startsWith(github.ref_name, 'release/')) }}
+        id: slack
+        uses: slackapi/slack-github-action@e28cf165c92ffef168d23c5c9000cffc8a25e117 # v1.24.0
+        with:
+          payload: |
+            {
+              "message": "One or more nightly integration tests have failed on ${{ github.ref_name }} branch. ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

Comment:
Do we want to modify this message to be more generic?

Reply:
ty!  fixed in bb5b6853555d1bf725fd70a1c5450082f58849a3

Reply:
Chatted over Zoom, we're thinking:
* Reuse this script to take advantage of author reporting and formatting
* Remove it from the job-level uses bc that will be incredibly noisy if we ever fix it
* Move to workflow-level in our change, update link and pass in the working webhook URL

Gonna pick up on this for a bit while John is in meetings